fix(examples): updated instructions for Python version in GitHub workflow example (#197)
fivetran/fivetran_connector_sdk: examples/workflows/github/README.md

- Clones the repository to the runner environment.

#### **Step 2: Set up Python**

> - Installs Python 3.11 for compatibility with Fivetran SDK.  

Reviewer: we should also add a link to our supported version

> - Installs Python 3.11 for compatibility with Fivetran SDK.  

Reviewer: @fivetran-dejantucakov, one of the key asks was to support a custom version via this example.

> The Python version here can technically be any supported version, but it’s preferable to keep it consistent with the version used in the `fivetran deploy` command (if provided) to avoid potential mismatches.

Reviewer 2: ```suggestion
You can use any supported Python version, but it’s preferable to keep it consistent with the version used in the `fivetran deploy` command (if provided) to avoid potential mismatches.
```
